core/teams/models.go

```diff
@@ -0,0 +1,52 @@
+package teams
```

Comment:
Not sure why this is not in the types package? I can imagine. there will be some ToProto / FromProto method used for snapshots or events etc.

Reply:
Alright.